data/examples_loading.py

```diff
@@ -0,0 +1,19 @@
+import json
+from typing import List
+from pydantic import BaseModel #this should not be needed here, this will get cleaned up when its moved to its own examples.py
+import os
+from models.models import Example
+
+class Examples(BaseModel): #do we need BaseModel? What does it do? 

```

Comment:
Clean comment, this just makes it a pydantic class rather than a normal python class